### DISCUSSION TITLE
Stack geocode: parameter confusion

### DISCUSSION THREAD

**Discussion by riyasmjgeo** [isce-framework/isce2#570] [Q&A]
I have prepared a sentinel stack and I wanted to geocode the products.
The program `geocodeIsce.py` requires REFERENCE and SECONDARY directories. I have doubts about these parameters. Should I give the reference according to the reference used in the stack or should I assign the reference of the pair I wanted to geocode?

**An example case:** 
I made a stack with 20200101 as the reference for the stack. The stack contains many pairs and 20200202_20200208 is a pair that I wanted to geocode. In this case, which is the REFERENCE to be supplied to `geocodeIsce.py`? Is it  20200101 or 20200202?

**Comment by EJFielding** [comment]
For the stack processing, the reference date for the stack should be used for the geocoding. That is the date that has the geometry data.
